react-components/src/apps/admin/AllStats/AppStatsComponents/NavBarTabs.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * Imported in AllStats.js. This function imports the files for all the tabs that are present under TabPanels
+ */
+
+import React from "react";
+import PropTypes from "prop-types";
+import { AppBar, Tabs, Tab, Typography, Box } from "@material-ui/core";
+import JobsTab from "./JobsTab";
+import AppsTab from "./AppsTab";
+import UsersTab from "./UsersTab";
+import ids from "./AllStatsIDs";
+import { getMessage, withI18N, build } from "@cyverse-de/ui-lib";
+import myMessagesFile from "./messages.js";
+
+function TabPanel(props) {
+    const { children, value, baseId, index, ...other } = props;
+
+    return (
+        <Typography
+            component="div"
+            role="tabpanel"
+            hidden={value !== index}
+            id={`nav-tabpanel-${index}`}
+            aria-labelledby={`nav-tab-${index}`}
+            {...other}
+        >
+            <Box p={3}>{children}</Box>
+        </Typography>
+    );
+}
+
+TabPanel.propTypes = {
+    children: PropTypes.node,
+    index: PropTypes.any.isRequired,
+    value: PropTypes.any.isRequired,
+};
+//
+// function a11yProps(index) {
+//     return {
+//         id: `nav-tab-${index}`,
+//         "aria-controls": `nav-tabpanel-${index}`,
+//     };
+// }
+
+function NavTabs(props) {
+    const classes = props;
+    const [value, setValue] = React.useState(0);
+
+    const handleChange = (event, newValue) => {
+        setValue(newValue);
+    };
+
+    function a11yProps(index) {
```

Comment:
Ah, but I still think it's worthwhile to keep the `aria-controls` key and value in the return. I think its value could be the same (it doesn't need a `baseId`).

Also, it looks like `baseId` is now used for the `id` value 👍, but `baseId` isn't getting passed into this function; only `index` is getting passed in.

Reply:
I would recommend uncommenting out the original function, but updating it to also accept a `baseId` parameter, and update its `id` field similar to this one.

Reply:
What is the use of `aria-controls`?
(Also I've made the necessary changes)

Reply:
Already discussed in Slack, but also posting here for future reference: https://www.w3.org/TR/wai-aria/